src/main/scala/org/broadinstitute/hail/driver/LinearRegressionCommand.scala

```diff
@@ -14,7 +14,7 @@ object LinearRegressionCommand extends Command {
   def description = "Test each variant for association using the linear regression model"
 
   class Options extends BaseOptions {
-    @Args4jOption(required = true, name = "-y", aliases = Array("--response"), usage = "Response sample annotation")
+    @Args4jOption(required = true, name = "-y", aliases = Array("--phenotype"), usage = "Response sample annotation")
```

Comment:
I changed these back to response.  I think it's better than phenotype, as the covariates are also phenotypes.